BUILD.bazel

```diff
@@ -55,39 +64,68 @@ js_library(
     visibility = ["//visibility:public"],
 )
 
+npm_package(
+    name = "npm_package",
+    srcs = [
+        "AUTHORS",
+        "BUILD.bazel",
+        "CODE_OF_CONDUCT.md",
```

Comment:
let's minify what we publish, I don't think we need things like CONTRIBUTING.md - if you want to contribute you'll come to our repo, not look through the distribution

Reply:
👍